src/valhalla_run_matrix.cc

```diff
@@ -45,20 +45,23 @@ std::string GetFormattedTime(uint32_t secs) {
 // Log results
 void LogResults(const bool optimize,
                 const valhalla::Options& options,
-                const valhalla::Matrix& matrix) {
-  LOG_INFO("Results:");
-  uint32_t idx1 = 0;
-  uint32_t idx2 = 0;
-  uint32_t nlocs = options.sources_size();
-  for (uint32_t i = 0; i < matrix.times().size(); i++) {
-    auto distance = matrix.distances().Get(i);
-    LOG_INFO(std::to_string(idx1) + "," + std::to_string(idx2) + ": Distance= " +
-             std::to_string(distance) + " Time= " + GetFormattedTime(matrix.times().Get(i)) +
-             " secs = " + std::to_string(distance));
-    idx2++;
-    if (idx2 == nlocs) {
-      idx2 = 0;
-      idx1++;
+                const valhalla::Matrix& matrix,
```

Comment:
this is from #4356 , never mind (or approve it here and I'll close the other one)